behaviors/TranslatablePage.php

```diff
@@ -0,0 +1,120 @@
+<?php namespace RainLab\Translate\Behaviors;
+
+use App;
+use RainLab\Translate\Classes\TranslatableBehavior;
+
+/**
+ * Translatable page model extension
+ *
+ * Usage:
+ *
+ * In the model class definition:
+ *
+ *   public $implement = ['@RainLab.Translate.Behaviors.TranslatablePage'];
+ *
+ */
+class TranslatablePage extends TranslatableBehavior
+{
+    protected $translatableAttributes = ['title', 'description', 'meta_title', 'meta_description'];
```

Comment:
I would remove this property and expect a `$translatable` property defined on `$model` like in most Translate Behaviors. We can add this dynamically in `Plugin.php` register method:

```
Page::extend(function($page) {
    $page->extendClassWith('RainLab\Translate\Behaviors\TranslatablePageUrl');
    $page->extendClassWith('RainLab\Translate\Behaviors\TranslatablePage');

    $page->addDynamicProperty('translatable', ['title', 'description', 'meta_title', 'meta_description']);
});
```

Another way is to just add this to model itself `Cms\Classes\Page` as seperate PR to CMS core.

This change will force update on all occurrences of `$this->translatableAttributes` to `$this->model->translatable`.

This will allow us for extensibility. When someone will extend CMS Page fields and will want to make fields translatable as well.